app/workers/send_user_invitation_worker.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class SendUserInvitationWorker < ApplicationJob
+  queue_as :default
+
+  ERRORS = [
+    OpenSSL::SSL::SSLError,
+    Net::SMTPAuthenticationError,
+    Net::SMTPFatalError,
+    Net::SMTPServerBusy,
+    Net::SMTPSyntaxError,
+    Net::SMTPUnknownError,
+    Net::SMTPUnsupportedCommand,
+    SocketError
+  ].freeze
+
+  def perform(invitation)
```

Comment:
catch the `ActiveJob::DeserializationError`

https://github.com/3scale/porta/blob/ff0b72315fcd58397aeac04aa254ad6c66376f52/app/workers/sphinx_indexation_worker.rb#L11-L13